core/common/src/main/java/alluxio/underfs/UnderFileSystemFactoryRegistry.java

```diff
@@ -69,6 +77,8 @@
 public final class UnderFileSystemFactoryRegistry {
   private static final Logger LOG = LoggerFactory.getLogger(UnderFileSystemFactoryRegistry.class);
 
+  // Key: JAR name
+  private static final Set<String> LOADED_EXTENSION_KEYS = new HashSet<>();
```

Comment:
or call it `LOADED_EXTENSION_JARS` and comment that we identify jars by absolute path